azure-pipelines.yml

```diff
@@ -35,12 +35,27 @@ pr:
     - "*"
 
 jobs:
+- job: checkout
+  displayName: Checkout repository
+  pool:
+    vmImage: ubuntu-18.04
+  steps:
+  - checkout: self
+    path: opentitan-repo
+  - bash: |
+      tar -C $(Pipeline.Workspace)/opentitan-repo -czf $(Pipeline.Workspace)/opentitan-repo.tar.gz .
```

Comment:
Does Azure automatically checkout the repo into `$(Pipeline.Workspace)/opentitan-repo`?

Reply:
It checks out the repo to `$(Pipeline.Workspace)` by default, but it doesn't put a parent folder around all the files. Instead, the `path` argument to the `checkout` step above tells Azure to checkout into `opentitan-repo` directory. Hence, the repo contents can be found in `$(Pipeline.Workspace)/opentitan-repo`.

Reply:
Cool, thanks for the explanation.